main.py

```diff
@@ -79,13 +81,25 @@ def shutdown() -> NoReturn:
 
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Reddit Video Maker Bot")
+    parser.add_argument("--post-id", help="Specify the post ID")
+    parser.add_argument("--run-many", type=int, help="Run the program multiple times")
+    parser.add_argument("--id")
```

Comment:
Python has a builtin library called Argparse. This adds CLI arguments to the program.
You can get the args from `add_argument` by using `argparse.parse_args()`.
The arguments also have a help param that add a mini-description to the args. So when you run the program and use the `help` param it would output the program description along with all of the param descriptions.

Reply:
No no no I'm aware of what argparse is. I was just asking why you added the unused ID flag

Reply:
Oh I'll fix it